draft-ietf-rtcweb-jsep.xml

```diff
@@ -1784,8 +1784,9 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           <t>
             <list style="symbols">
 
-              <t>The profile in any "m=" line in any answer MUST
-              exactly match the profile provided in the offer.</t>
+              <t>The profile in any "m=" line in any answer generated 
```

Comment:
The text above already makes it clear that this is about the JSEP implementation's processing of an offer, so this seems redundant.

However, I could live with "in any generated answer"